tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -66,12 +89,18 @@ echo "Waiting 2m for instance to initialize..."
 sleep 2m
 
 WORKLOAD=grpc_aws_experiment_remote.sh
-REMOTE_SCRIPT_FAILURE=0
-echo "Copying to remote instance..."
-scp -i ~/.ssh/temp_client_key -r github/grpc ubuntu@$IP:
+REMOTE_SCRIPT_EXITCODE=0
+echo "Copying workspace to remote instance..."
+# use rsync over ssh since it's much faster than scp
+time rsync -e "ssh -i ~/.ssh/temp_client_key" -a github/grpc ubuntu@$IP:~/workspace
 echo "Beginning CI workload..."
-ssh -i ~/.ssh/temp_client_key ubuntu@$IP "uname -a; ls -l; bash grpc/tools/internal_ci/linux/$WORKLOAD" || REMOTE_SCRIPT_FAILURE=$?
+ssh -i ~/.ssh/temp_client_key ubuntu@$IP "uname -a; cd ~/workspace; ls -l; bash grpc/tools/internal_ci/linux/$WORKLOAD" || REMOTE_SCRIPT_EXITCODE=$?
```

Comment:
We can drop ls -l, which was for debugging only.

Reply:
sg, we can do in a followup.